sinfonier-web/app/views/tags/topologies/_topology.html

```diff
@@ -31,7 +31,7 @@ <h2 class="title"><a id="topology-${_topology.getId()}-a-title" href="/topologie
         </div>
       #{/if}
 
-      <div class="col-lg-12" id="log">
+      <div class="col-lg-12 topology-log" id="log"  data-id="${_topology.getId()}" data-log-url="@{Topologies.log(_topology.getId())}" >

```

Comment:
This an invalid id, this html snippet is used in the list of topologies, so you will n topologies with the same id.